packages/apktool.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,15 +4,30 @@ Import-Module vm.common -Force -DisableNameChecking
 try {
   $toolName = 'apktool'
   $category = 'Java & Android'
-  $shimPath = 'bin\apktool.exe'
+  $rawToolPath = Join-Path ${Env:RAW_TOOLS_DIR} "$toolName"
+
+  # Download apktool.bat
+  $wrapperPath = Join-Path $rawToolPath "$toolName.bat"
+  $wrapperSource = 'https://raw.githubusercontent.com/iBotPeaches/Apktool/master/scripts/windows/apktool.bat'
```

Comment:
I can use that commit instead. 👍 

I just followed what was on the 'Install Guide' in their documentation for Windows and how to run it: https://apktool.org/docs/install/

I assumed that the `.bat` file is something that does proper setup for `apktool` and wanted to follow what they recommend for installation, but if you think we should maybe just call `apktool` directly using our own shortcut, I can look into that instead.

Reply:
Updated so that it uses the commit!

Just need to know how to go forward with whether or not we should use the `.bat` (as suggested by the tool author's install guide) or if we should just do everything in our own shortcut instead.

Reply:
I was a bit confused as I think the metapackage that we used to use did it differently. But I think **we should do it as suggested by the too author's guide and test it** to ensure that it works.